content/07.pathogenesis.md

```diff
@@ -46,8 +46,12 @@ One severe example includes reports of cytokine storm-like responses in patients
 #### Reproduction Number and Dynamics of Transmission
 
 Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
-The basic reproduction number, R~0~, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
+In models the quantities of basic reproduction (or reproductive) number and the timescale of infection have a large influence on the population level dynamics of epidemics.
+The basic reproduction number, R~0~, is the expected number of new (secondary) infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
+R~0~ is a unitless number which in models is a combination of parameters principally related to the transmission rate (rate of infection transmitting interactions for an infectious individual) and the infectious period.
+When R~0~ is greater than 1, there is potential for an epidemic to grow.
 The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~, for example as some fraction of the population becomes infected, or as interventions are put into place.
+When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that's infectious increases); when R~t~ is smaller than 1, the proportion of the population that's infectious decreases.
```

Comment:
```suggestion
As for R~0~, if R~t~ is greater than 1, then the number of infected individuals increases over time; for R~t~ less than 1 the number of infected individuals decreases over time. Implementing measures that decrease or maintain R~t~ below 1 is thus necessary for containing an outbreak. 
```

Reply:
I think may be important to add a caveat here about placing too much emphasis on real-time estimates of Rt - the use of these to make policy decisions is worrying as I do not think the concept is well-understood, plus short-window averages are likely to contain a lot of noise! Plus testing delays, etc etc. eg. [this](https://news.sky.com/story/coronavirus-virus-could-be-wiped-out-in-london-in-weeks-as-rate-of-infection-is-slowing-say-researchers-11988579) misleading headline.

Reply:
I also am more experienced in mechanistic modeling rather than lots of real-time Rt estimates but this is what is reviewed so far & whoever wrote it did a good job of putting a variety of articles, so I was trying to complement this by putting clearer definitions earlier on.

Reply:
@alavendelm there's a very short (~3 page) science paper which may have some points slightly related at least in general  to your point of possible misinterpretations (e.g. when science and its application isn't understood/explained clearly enough or when nuances or math are skipped). Regardless, you may find it interesting. I'll add a brief summary as an issue but really the whole 3 pages are dense with interesting information. Article/Perspectives: Modeling infectious disease dynamics. The spread of the coronavirus SARS-CoV-2 has predictable features.  S. Cobey, Science 10.1126/science.abb5659 (2020)

Reply:
I think these sentence incorporating some of this suggestion would be good, with the 2nd suggested sentence by @alavendelm fully solidifying to the readers early in the section that the goal is to get R below 1 (which was the main thing I was going for & maybe addresses in part @rdvelazquez Issue topic on clarifying R):
"If R~t~ is greater than 1, then the number of infected individuals increases over time; for R~t~ less than 1 the number of infected individuals decreases over time.
Implementing measures that decrease or maintain R~t~ below 1 is thus necessary for containing an outbreak."
Again, it would be better at this point if someone fixed/added this so I don't make a learner's mistake on this somewhat complicated pull request.